cypress/constants/routes/routes.js

```diff
@@ -0,0 +1,3 @@
+export const routes = {
+    path: 'https://spinkart.neetoauth.net/login',
```

Comment:
This would be sufficient as base url is set and please give meaningful variable names.
```suggestion
    loginPath: '/login',
```